From be2fdfdada6d419b4b80de2449e845d323516f72 Mon Sep 17 00:00:00 2001
From: Tero Paloheimo <tero.paloheimo@iki.fi>
Date: Tue, 1 Jan 2019 21:53:24 +0200
Subject: [PATCH] Fix implicit declaration warning of mkostemp

mkostemp() requires the _GNU_SOURCE feature test macro.
See man 3 mkostemp.
---
 pam_yubico.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pam_yubico.c b/pam_yubico.c
index 2c6c5d9..1dbad63 100644
--- a/pam_yubico.c
+++ b/pam_yubico.c
@@ -28,6 +28,8 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#define _GNU_SOURCE
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
